setup.sh

```diff
@@ -58,90 +115,80 @@ else
 fi
 
 # Check if the backend module can be removed
+if ! whiptail --backtitle "New CARDS repository setup" --title "Modules setup" --yesno "Does the project have backend code?" 8 78
+then
+  git rm -rf backend
+  sed -i -e '/backend/,+3d' feature/src/main/features/feature.json
+  sed -i -e '/backend/d' pom.xml
+fi
 
-echo
-echo "Does the project have backend code?"
-HAS_BACKEND=maybe
-until [[ $HAS_BACKEND == "yes" || $HAS_BACKEND == "no" ]]
-do
-  read -e -p "[yes|no] " -i "no" HAS_BACKEND
+./get_cards_platform_jars.sh ${CARDS_VERSION}
```

Comment:
Exit here with an error if `get_cards_platform_jars.sh` fails.

```suggestion
./get_cards_platform_jars.sh ${CARDS_VERSION} || exit -1
```